### PR TITLE
Add the urls of the homebankings, and add url at schema

### PR DESCRIPTION
**Pull request by SantiDepetris**
Url for homebankings added
https://github.com/eldolar/web/issues/5

### CODE REVIEW

File: src/services/fetchers/supervielle.ts [hunk 1]
```diff
@@ -25,13 +25,15 @@ class Supervielle {
 
       return {
         name: 'Supervielle',
+        url: 'https://bit.ly/2TfvDaK',
```
Comment (edudepetris) [inline src/services/fetchers/supervielle.ts]: @SantiDepetris 
this link is pointing to `empresas homebanking` we should point out https://personas.supervielle.com.ar/

File: src/services/fetchers/nacion.ts [hunk 2]
```diff
@@ -21,15 +21,17 @@ class Nacion {
     
       return {
         name: 'Nación',
+        url: "https://bit.ly/2FWkJE4"
```
Comment (edudepetris) [inline src/services/fetchers/nacion.ts]: @SantiDepetris 
missing `,` at the end

Comment (edudepetris) [reply src/services/fetchers/nacion.ts]: We need to update the URL to point here https://hb.redlink.com.ar/bna/login.htm

File: src/services/fetchers/nacion.ts [hunk 3]
```diff
@@ -21,15 +21,17 @@ class Nacion {
...
         buy: parseFloat(buy),
-        sell: parseFloat(sell)
+        sell: parseFloat(sell),
       }
     })
     .catch(err => {
       return {
         name: 'Nación',
+        url: "https://bit.ly/2FWkJE4"
```
Comment (edudepetris) [inline src/services/fetchers/nacion.ts]: missing `,`

**Review by edudepetris — CHANGES_REQUESTED** [review]
Some changes addressing on comments.

Plus: Could you add the new field `url` to each test.

For that, it'll be necessary to add the `url` to `BankOption`, and then to each test.

Eg.:

```diff
diff --git a/src/services/bankInterfaces.ts b/src/services/bankInterfaces.ts
index 58c194c..5ef29e9 100644
--- a/src/services/bankInterfaces.ts
+++ b/src/services/bankInterfaces.ts
@@ -1,5 +1,6 @@
 export default interface BankOptions {
   name: string,
   buy?: number,
-  sell?: number
+  sell?: number,
+  url: string,
 }
diff --git a/test/services/fetchers/bancor.spect.ts b/test/services/fetchers/bancor.spect.ts
index 174a76f..d7f5255 100644
--- a/test/services/fetchers/bancor.spect.ts
+++ b/test/services/fetchers/bancor.spect.ts
@@ -18,6 +18,7 @@ describe('fetchers/bancor', () => {
     expect(result.name).to.equal('Bancor');
     expect(result.buy).to.equal(44.4);
     expect(result.sell).to.equal(46.9);
+    expect(result.url).to.be.a('string');
   })
```

Also we need to update the api's documentation:
|api-doc|
|-|
|Screen Shot 2020-10-23 at 7 58 09 PM|

You can do it, adding that here https://github.com/SantiDepetris/api/blob/eca6bc4df5cd8b8a0a55f3368881b0d876df61cd/src/routers/exchangeRouter.ts#L21

File: src/services/fetchers/nacion.ts (hunk 3 above)
Comment (SantiDepetris) [reply src/services/fetchers/nacion.ts]: Done

File: src/services/fetchers/nacion.ts (hunk 2 above)
Comment (SantiDepetris) [reply src/services/fetchers/nacion.ts]: Done

File: src/services/fetchers/supervielle.ts (hunk 1 above)
Comment (SantiDepetris) [reply src/services/fetchers/supervielle.ts]: Done

File: src/services/fetchers/nacion.ts [hunk 4]
```diff
@@ -21,15 +21,17 @@ class Nacion {
...
+        url: "https://bit.ly/35yOxzf",
         buy: parseFloat(buy),
-        sell: parseFloat(sell)
+        sell: parseFloat(sell),
       }
     })
     .catch(err => {
       return {
         name: 'Nación',
+        url: "https://bit.ly/35yOxzf"
```
Comment (edudepetris) [inline src/services/fetchers/nacion.ts]: @SantiDepetris missing `,`

Comment (SantiDepetris) [reply src/services/fetchers/nacion.ts]: Done

**Comment by edudepetris** [comment]
@SantiDepetris Thanks 😊 for your contribution 💙

**Review by edudepetris — APPROVED** [review]

**Pull request merged**